### DISCUSSION TITLE
Showcase of apps using CapacitorJS

### DISCUSSION THREAD

**Discussion by zakuru** [ionic-team/capacitor#5939] [Q&A]
Hi all,
First thanks for the amazing work achieved by the team and the stunning results that CapacitorJS brings,
Is it possible to have a list / showcase of mobile apps using CapacitorJS ? either Android or Ios ...
That would help a lot !

**Comment by jeremejazz** [comment]
From what I know, Obsidian is using CapacitorJS.
